src/filters/entropy.meter/pre.filters/email.addresses.js

```diff
@@ -1,4 +1,4 @@
 /**
  * Skip all email addresses because they generate high entropy results.
  */
-module.exports = s => !s.match(/^[a-z0-9+_.-]+@[a-z._-]+\.[a-z]+$/i);
+module.exports = s => !s.match(/^[\w\d+.-]+@[\w\d.-]+\.[\w]+$/i);
```

Comment:
I did my best to test the regex here: https://emailregex.com/
Seems like it worked for all of the examples that I had as well.

Reply:
Thanks!